compiler/frontend/src/org/jetbrains/kotlin/frontend/di/injection.kt

```diff
@@ -79,6 +80,8 @@ fun StorageComponentContainer.configureModule(
     analyzerServices.platformConfigurator.configureModuleComponents(this)
     analyzerServices.platformConfigurator.configureModuleDependentCheckers(this)
 
+    useInstance(TypeAttributeTranslators(moduleContext.project))
```

Comment:
It's better to use `useImpl<TypeAttributeTranslators>()` here. `project` will be injected automatically